tests/integration/rapidgen/rapidgen.go

```diff
@@ -127,9 +127,7 @@ var (
 		GenType(&disttypes.MsgWithdrawDelegatorReward{}, &distapi.MsgWithdrawDelegatorReward{}, GenOpts),
 		GenType(&disttypes.MsgWithdrawValidatorCommission{}, &distapi.MsgWithdrawValidatorCommission{}, GenOpts),
 		GenType(&disttypes.MsgSetWithdrawAddress{}, &distapi.MsgSetWithdrawAddress{}, GenOpts),
-		GenType(&disttypes.MsgFundCommunityPool{}, &distapi.MsgFundCommunityPool{}, GenOpts),
```

Comment:
Shouldn't not add pool messages somewhere here?

Reply:
adding pool messages here is resulting in `aminojson_test` failure, since there is no amino support. Should I remove them?

Reply:
Ah okay, makes sense yeah!